src/Plugin/Commerce/ShippingMethod/USPS.php

```diff
@@ -95,13 +153,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
 
     $form['conditions'] = [
       '#type' => 'details',
-      '#title' => $this->t('usps rate conditionings'),
-      '#description' => $this->t('setting when USPS Rates are excluded'),
+      '#title' => $this->t('USPS rate conditions'),
     ];
 
     $form['conditions']['conditions'] = [
       '#type' => 'checkboxes',
       '#title' => $this->t('Exclude USPS Rates'),
```

Comment:
I'm actually not sure if this is correct because the USPS service keys we get back are actually integers and the keys here are string. So, I'm not sure about this one.

Reply:
A couple of questions:

* Why do we not use services i.e. register the services in the shipping method's annotations, like we do in UPS?
* Why do we allow excluding services instead of including, like we do in UPS and other modules?

Reply:
To be honest, I have no idea. The [initial work](https://github.com/mattwebdev/commerce_usps) was setup this way by Matt. We'd have to get in touch with him to understand the actual reason. I didn't want to change it as I wasn't sure of the reason.